client/components/events/EventViewToggle/EventViewToggle.tsx

```diff
@@ -88,13 +89,13 @@ export const EventViewToggle = ({ filteredEvents }: EventViewProps) => {
 
 interface ActiveViewProps {
   activeView: View,
-  filteredEvents: Array<Event>
+  filteredEvents: FilteredEvents
 }
 
-const ActiveView = ({activeView, filteredEvents}: ActiveViewProps) => {
+const ActiveView = ({activeView, filteredEvents }: ActiveViewProps) => {
   
   const getActiveView = () => {
```

Comment:
We're defining this lambda and then calling it right away; can we just inline it?